src/Auth0Client.ts

```diff
@@ -668,8 +668,17 @@ export default class Auth0Client {
       throw new Error('Invalid state');
     }
 
+    const {
+      scope,
+      audience,
+      redirect_uri,
+      ignoreCache,
+      ...customOptions
+    } = options;
+
     const tokenResult = await oauthToken(
       {
+        ...customOptions,
```

Comment:
You'll need to add custom options to `TokenEndpointOptions`

Reply:
Good catch